src/modify.js

```diff
@@ -92,20 +95,55 @@ function rewriteAllFields(schema, configCallback, context) {
   });
 }
 
+function createFields(schema, fieldsConfig) {
+  if (!fieldsConfig) return { warnings: null };
+
+  const warnings = [];
+  const fieldsToCreate = Object.entries(fieldsConfig);
+
+  fieldsToCreate.forEach(([shortPath, fieldAttrs]) => {
+    const fieldPath = shortToFullPath(shortPath);
+
+    if (fieldAttrs.properties) {
+      // Recursive to nested fields...
+      const result = createFields(get(schema.properties, fieldPath), fieldAttrs.properties);
+      warnings.push(result.warnings);
+    }
+
+    const fieldInSchema = get(schema.properties, fieldPath);
+    const { properties, ...otherAttrs } = fieldAttrs;
+
+    if (fieldInSchema && otherAttrs) {
```

Comment:
Why do we need to check for `otherAttrs` here?

Reply:
This one is tough to explain. Basically, we have 2 types of changes to fieldsets:
- the fieldset itself (eg the "address" title — those are is "otherAttrs")
- a field inside the fieldset. (eg address.street — those are the "properties").

And the initial implemented needed this, so we could distinguish when to ignore it or not. 

Then I refactored the order of operations in this function to not be needed but forgot to add this additional check. Removed!